src/components/consul/detail/ConsulQuestionsListItem.js

```diff
@@ -58,7 +57,7 @@ export const ConsulQuestionsListItem = ({ questionItem, onRefresh, token, disabl
           ]}
         >
           <RegularText placeholder={!disabled}>{item.title}</RegularText>
-        </Touchable>
+        </TouchableOpacity>
```

Comment:
there is also a `<Touchable` including `<RegularText`... in `ConsulCommentListItem`. Does this need adjustments as well?